.ci/test-windows.ps1

```diff
@@ -66,7 +66,7 @@ if ($env:TASK -eq "swig") {
 conda init powershell
```

Comment:
Why not to bump version in Appveyor?
https://github.com/microsoft/LightGBM/blob/3654ecaaa1a33628dd2b1cc936c3f1cfd31079a4/.appveyor.yml#L6

Reply:
I was thinking that because we use Appveyor to test such an old version of VS Code:

https://github.com/microsoft/LightGBM/blob/3654ecaaa1a33628dd2b1cc936c3f1cfd31079a4/.appveyor.yml#L3

That it should stay frozen on an old Python version too, until supporting that older Python version on Appveyor breaks. Given that, do you think we should make increasing this version to `3.9` part of this update?

Reply:
OK, makes sense! I don't mind leaving 3.8.